src/js/ripple/transaction.js

```diff
@@ -466,7 +466,7 @@ Transaction.prototype.sign = function() {
   }
 
   var key = seed.get_key(this.tx_json.Account);
-  var sig = key.sign(hash, 0);
+  var sig = key.sign(hash);
```

Comment:
That function only takes one parameter: https://github.com/clark800/ripple-lib/blob/c0849e50b3db09429f3b1688fd8e0fd340433ff8/src/js/ripple/keypair.js#L94